embedding/csharp/Tizen.Flutter.Embedding/NUIFlutterView.cs

```diff
@@ -0,0 +1,205 @@
+﻿// Copyright 2022 Samsung Electronics Co., Ltd. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+using System;
+using System.Collections.Generic;
+using System.Diagnostics;
+using System.Reflection;
+using System.Runtime.InteropServices;
+using Tizen.NUI;
+using Tizen.NUI.BaseComponents;
+using static Tizen.Flutter.Embedding.Interop;
+
+namespace Tizen.Flutter.Embedding
+{
+    /// <summary>
+    /// Displays a Flutter screen in a Tizen application.
+    /// </summary>
+    public class NUIFlutterView : ImageView
+    {
+        public NUIFlutterView()
+        {
+            _viewSize = Size2D;
+        }
```

Comment:
`_viewSize` will be overwrriten in `RunEngine` so setting the value here seems redundant.

Also all other classes have property and member definitions before the constructor definition so please reorder properly.

Reply:
As you said, we don't need this initialization. I removed the constructor.